--- a/locale/Makefile
+++ b/locale/Makefile
@@ -81,6 +81,12 @@
 # The path to the compiled binary locale archive or compiled locales,
 # along with the parent path to the source locales and source
 # charmaps.
+complocaledir = /kip/usr/lib/locale
+i18ndir = /kip/usr/share/i18n
+rtld-LDFLAGS = -Wl,--dynamic-linker=/kip/lib/ld.so.1
+LDFLAGS-locale = -Wl,-rpath,/kip/lib
+LDFLAGS-localedef = -Wl,-rpath,/kip/lib
+
 localepath = "$(complocaledir):$(i18ndir)"
 
 # -Iprograms doesn't really belong here, but this gets it at the head
